kafka/src/test/java/com.expedia.adaptivealerting.kafka/KafkaAnomalyDetectorManagerTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.kafka;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.fail;
+
+/**
+ * @author Willie Wheeler
+ */
+public class KafkaAnomalyDetectorManagerTest {
+    
+    @Test
+    @Ignore
+    public void testDummy() {
```

Comment:
Do we want to keep this? Is it a placeholder?

Reply:
Yeah, just a placeholder to avoid JUnit complaining about no tests.